vyper/compiler/README.md

```diff
@@ -25,13 +25,15 @@ The compilation process has five broad phases:
 
 1. In [`vyper.ast`](../ast), the source code is parsed and converted to an
 abstract syntax tree.
-2. The [`GlobalContext`](../parser/global_context.py) object is generated from the
+2. In [`vyper.ast.folding`](../ast/folding.py), literal Vyper AST nodes are
+evaluated and replaced with the resulting values.
+3. The [`GlobalContext`](../parser/global_context.py) object is generated from the
 Vyper AST, analyzing and organizing the nodes prior to LLL generation.
-3. In [`vyper.parser.parser`](../parser/parser.py), the contextualized nodes are
```

Comment:
You can just use `1.` for all items in a markdown numbered list and it turns out okay. This way you don't have to modify them every time.

Reply:
Voodoo!